src/main/java/org/folio/marccat/resources/shared/ConversionFieldUtils.java

```diff
@@ -411,6 +408,39 @@ public static FixedField getDisplayValueOfMaterial(final FixedField fixedField,
     return fixedField;
   }
 
+  /**
+   * Return a display value for the material description.
+   *
+   * @param fixedField     the fixedField to populate.
+   * @param formOfMaterial the form of material.
+   */
+  public static FixedField getAuthorityDisplayValueOfMaterial(final FixedField fixedField) {
+    StringBuilder sb = new StringBuilder();
+
+    sb.append(fixedField.getDateEnteredOnFile());
```

Comment:
Do we know for sure if `fixedField != null`??  Looks like we get it from the body of the POST request.

Reply:
I add the control of this param. But I think that it never could be null.